core/client/src/test/java/alluxio/client/block/stream/NettyPacketReaderTest.java

```diff
@@ -0,0 +1,256 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.block.stream;
+
+import alluxio.Constants;
+import alluxio.EmbeddedChannels;
+import alluxio.client.file.FileSystemContext;
+import alluxio.network.protocol.RPCProtoMessage;
+import alluxio.network.protocol.databuffer.DataBuffer;
+import alluxio.network.protocol.databuffer.DataNettyBufferV2;
+import alluxio.proto.dataserver.Protocol;
+import alluxio.util.CommonUtils;
+import alluxio.util.WaitForOptions;
+import alluxio.util.io.BufferUtils;
+
+import com.google.common.base.Function;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.channel.embedded.EmbeddedChannel;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import java.net.InetSocketAddress;
+import java.util.Random;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({FileSystemContext.class})
+public final class NettyPacketReaderTest {
+  private static final int PACKET_SIZE = 1024;
```

Comment:
I get PACKET_SIZE from configuration  on NettyDataServerReadHandlerTest.java because it has be to consistent since the the reader (non-test code) there reads the data from a file (prepared by the test) with packet size derived from the config. 

On the contrary, NettyPacketReader implementation doesn't need to read packet size from the config as it simply consumes what ever the server sends.